### PR TITLE
[mp] aws ecs docs & architecture diagram

### PR DESCRIPTION
**Pull request by mattppal**
# Summary

This creates a page with an architecture diagram and walkthrough of our AWS ECS deployment.

# Tests
 
Mintlify, local.

cc:

### CODE REVIEW

**Comment by wangxiaoyou1993** [comment]
What's "local-auth.png" for?

File: docs/production/deploying-to-cloud/aws/resources.mdx [hunk 1]
```diff
@@ -687,14 +706,14 @@ Terraform will perform the following actions:
       + ingress                = [
           + {
               + cidr_blocks      = [
-                  + "0.0.0.0/0",
+                  + "98.234.38.28/32",
```
Comment (wangxiaoyou1993) [inline docs/production/deploying-to-cloud/aws/resources.mdx:709]: Don't put your IP here.

File: docs/production/deploying-to-cloud/aws/resources.mdx [hunk 2]
```diff
@@ -597,7 +561,7 @@ Terraform will perform the following actions:
             },
           + {
               + cidr_blocks      = [
-                  + "11.111.11.111/32",
+                  + "98.234.38.28/32",
```
Comment (wangxiaoyou1993) [inline docs/production/deploying-to-cloud/aws/resources.mdx:564]: Replace your IP with faked one.

File: docs/production/deploying-to-cloud/aws/resources.mdx [hunk 3]
```diff
@@ -584,7 +548,7 @@ Terraform will perform the following actions:
       + ingress                = [
           + {
               + cidr_blocks      = [
-                  + "98.207.95.145/32",
+                  + "98.234.38.28/32",
```
Comment (wangxiaoyou1993) [inline docs/production/deploying-to-cloud/aws/resources.mdx:551]: Same here.

File: docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx [hunk 4]
```diff
@@ -3,5 +3,53 @@ title: "AWS Elastic Container Service"
 sidebarTitle: "AWS ECS"
 ---
 
-You can refer to the instructions found [here](/production/deploying-to-cloud/aws/setup).
```
Comment (wangxiaoyou1993) [inline docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx:6]: This is not the right doc. This doc is for multi-dev environment on cloud.

You should put the architecture doc in this folder: https://github.com/mage-ai/mage-ai/tree/master/docs/production/deploying-to-cloud/aws

File: docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx [hunk 5]
```diff
@@ -3,5 +3,53 @@ title: "AWS Elastic Container Service"
...
+
+We use the following AWS services in our deployment:
+
+## Elastic Container Service (ECS)
+
+Amazon ECS is a service provided by AWS for managing and deploying containerized applications. The ECS service in Terraform is used to define and configure the ECS clusters and services. 
+
+ECS allows you to run containers at scale, and the Terraform configuration for ECS service includes specifying the task definition, desired task count, launch type, and other settings related to container orchestration.
+
+The Mage application runs as multiple containers in an Amazon ECS instance.
```
Comment (wangxiaoyou1993) [inline docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx:23]: You can say ECS service instead of ECS instance.

**Comment by mattppal** [comment]
> What's "local-auth.png" for?

This was a broken link from another doc that I noticed— shows where to look for a key when authenticating.

File: docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx (hunk 5 above)
Comment (mattppal) [reply docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx:23]: ECS stands for elastic container service, so I think that's redundant

Comment (wangxiaoyou1993) [reply docs/developing-in-the-cloud/cloud-dev-environments/aws-ecs.mdx:23]: No, that's not redundant. https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html

File: docs/production/deploying-to-cloud/aws/aws-ecs.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
+---
+title: "AWS Elastic Container Service"
```
Comment (wangxiaoyou1993) [inline docs/production/deploying-to-cloud/aws/aws-ecs.mdx:2]: Can you name the doc "AWS ECS deployment architecture"? The sidebarTitle can be "Architecture"

**Review by wangxiaoyou1993 — COMMENTED** [review]
Can you add the new doc to mint.json?

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**